apps/channels/models.py

```diff
@@ -94,14 +94,3 @@ def _set_telegram_webhook(experiment_channel: ExperimentChannel):
 
     tele_bot.set_webhook(webhook_url, secret_token=settings.TELEGRAM_SECRET_TOKEN)
     tele_bot.set_my_commands(commands=[types.BotCommand(ExperimentChannel.RESET_COMMAND, "Restart chat")])
-
```

Comment:
Thanks for this. Was planning on doing this tomorrow, but you beat me to it :+1: